Apps/Inc/PID.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 UT Longhorn Racing Solar */
+/** PID.h
+ * File contains PID loop function
+ */
+
+#ifndef PID_H
+#define PID_H
+
+#include "common.h"
+
+/**
+ * @brief Gives fan speed based on Average temperature of pack and past error values
+ * @param InputTemp - current temperature
+ * @param DesiredTemp - desired temperature
+ * @return FanSpeed: 0-4000 PWM
+ */
+int32_t PID_Output(int32_t InputTemp, int32_t DesiredTemp);
+
+#endif
```

Comment:
If this is going to just be used for temperature, then just include it in temperature.h. If it is going to be used for other PID later down the line, you should make it a more general function.